CoreBluetoothMock/CBMPeripheralSpecDelegate.swift

```diff
@@ -251,155 +251,113 @@ public protocol CBMPeripheralSpecDelegate {
 public extension CBMPeripheralSpecDelegate {
 
     func reset() {
-        // Empty default implementation
+        // No-op.
     }
 
-    func peripheralDidReceiveConnectionRequest(_ peripheral: CBMPeripheralSpec)
-        -> Result<Void, Error> {
-            return .success(())
+    func peripheralDidReceiveConnectionRequest(_ peripheral: CBMPeripheralSpec) -> Result<Void, Error> {
+        .success(())
     }
     
     func peripheral(_ peripheral: CBMPeripheralSpec, didDisconnect error: Error?) {
         assert(peripheral.virtualConnections == 0)
     }
     
-    func peripheral(_ peripheral: CBMPeripheralSpec,
-                    didReceiveServiceDiscoveryRequest serviceUUIDs: [CBMUUID]?)
-        -> Result<Void, Error> {
-            return .success(())
```

Comment:
This may break compatibility with some old Swift version, when the return was needed. We specify in CocoaPods _Podfile_ all Swift versions that it works with, including 4.2.